src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(.\fonts\Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(.\fonts\Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir;
+}
+
+.search-field {
+  display: flex;
+  align-items: center;
+}
+
+.search-field:focus {
+  background: linear-gradient(180deg, #FFFFFF 0%, #F6F6F7 100%);
+  border: 1px solid #E1E7ED;
+  border-radius: 3px;
+}
+
+.search-icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  margin-left: 26px;
+  z-index: 20;
+}
+
+.search-input {
+  background: #FFFFFF;
+  border: 1px solid #E1E7ED;
+  box-shadow: 0px 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 26px;
+  height: 70px;
+  width: 100%;
+  font-family: Avenir;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3D4E61;
+  border: 0px;
+  position: relative;
+}
+
+.search-input:hover {
+  border: 1px solid #E1E7ED;
+  box-shadow: 0px 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-input:focus {
+  outline: none;
+  background: linear-gradient(180deg, #FFFFFF 0%, #F6F6F7 100%);
+  border: 1px solid #E1E7ED;
+  border-radius: 3px;
+  filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+  font-weight: 700;
+}
+
+.search-input::before {
+  content: "апцукп";
+  z-index: 999;
+}
```

Comment:
Better to wrap input in label and set ::before on label.
styles for label will be something like that
```
    position: relative;
    display: block;
    width: 100%;
```

Reply:
for pseudo element required
```
    position: absolute;
    content: "";
    background-image: url(...);
    width: ...;
    height: ...;
    top: ...;
    left: ...;